Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticationPanelClient.h

```diff
@@ -34,27 +34,23 @@
 #import <wtf/WeakObjCPtr.h>
 #import <wtf/WeakPtr.h>
 
-namespace WebKit {
-class WebAuthenticationPanelClient;
-}
-
-namespace WTF {
-template<typename T> struct IsDeprecatedWeakRefSmartPointerException;
-template<> struct IsDeprecatedWeakRefSmartPointerException<WebKit::WebAuthenticationPanelClient> : std::true_type { };
-}
-
 @class _WKWebAuthenticationPanel;
 @protocol _WKWebAuthenticationPanelDelegate;
 
 namespace WebKit {
 
 class WebAuthenticationPanelClient final : public API::WebAuthenticationPanelClient, public CanMakeWeakPtr<WebAuthenticationPanelClient> {
 public:
-    WebAuthenticationPanelClient(_WKWebAuthenticationPanel *, id <_WKWebAuthenticationPanelDelegate>);
+    static Ref<WebAuthenticationPanelClient> create(_WKWebAuthenticationPanel *, id<_WKWebAuthenticationPanelDelegate>);
 
-    RetainPtr<id <_WKWebAuthenticationPanelDelegate> > delegate();
+    void ref() const { RefCounted::ref(); }
```

Comment:
`final`

Reply:
If moving back to previous, then this cannot be final because ref() is not virtual in RefCounted.